### ISSUE TITLE
go-xcat to install stable version is broken

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#6349]
When trying to install stable version of xcat using go-xcat, it fails because the links are not there..  So this is a devops problem in that we don't have the correct scripts in place to generate this when we release ...    but not sure when we will get around to it...  so creating this as a placeholder for now ... 

## Workaround

Instead of using`--xcat-version=stable` use the major release to get the latest stable. 

For latest 2.14.x, use `-x 2.14` and  it will install the latest stable release  

## data

```
[root@c910f3u25-fs4 yum.repos.d]# /tmp/go-xcat --xcat-version=stable install
Operating system:   linux
Architecture:       ppc64le
Linux Distribution: rhel
Version:            7.6
go-xcat Version:    1.0.40


Reading repositories ...... failed

...
...
http://xcat.org/files/xcat/repos/yum/stable/xcat-core/repodata/repomd.xml: [Errno 14] HTTP Error 404 - Not Found
...
...
failure: repodata/repomd.xml from xcat-core: [Errno 256] No more mirrors to try.
http://xcat.org/files/xcat/repos/yum/stable/xcat-core/repodata/repomd.xml: [Errno 14] HTTP Error 404 - Not Found
...
...
0 metadata files removed
0 sqlite files removed
0 metadata files removed

Boo-boo
=======

Something went wrong. :(

Please check log file `/tmp/go-xcat.log' for more details.
```

Screen capture shows it's not there

![image](https://user-images.githubusercontent.com/10049070/58960870-074eb480-8776-11e9-9164-229dce36b68e.png)

**Comment by neo954** [comment]
Use `--xcat-version=latest` or omit the `--xcat-version` command line argument will make `go-xcat` get the latest stable version.

**Comment by gurevichmark** [comment]
Maybe `go-xcat` should recognize the `stable` flag and install the `latest` version.
OR
Maybe during a release process a `stable` link is created so that `-xcat-version=stable` works

**Comment by gurevichmark** [comment]
Resolved by #6692
